integration-tests/bats/doltpy.bats

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bats
+load $BATS_TEST_DIRNAME/helper/common.bash
+
+setup() {
+    setup_common
+    dolt sql <<SQL
+CREATE TABLE foo (
+  a BIGINT NOT NULL,
+  b BIGINT,
+  PRIMARY KEY (a)
+);
+INSERT INTO foo VALUES (0,0), (1,1);
+SELECT DOLT_COMMIT('-am', 'Initialize table');
+SQL
+}
+
+teardown() {
+    assert_feature_version
+    teardown_common
+}
+
+# NOTE: if these break on a release that's OK, just tell Max/Oscar
+
+# already covered by other tests:
+# "select * from dolt_branches"
+# "SELECT * FROM `{table}` ASOF '{commit}'"
+
+# verifying the output comparison to dolt log would be nice
+@test "doltpy: Dolt.log" {
+    run dolt sql -r csv <<SQL
+SELECT
+    dc.commit_hash as commit_hash,
+    dca.parent_hash as parent_hash,
+    committer as committer,
+    email as email,
+    date as date,
+    message as message
+FROM
+    dolt_log AS dc
+    LEFT OUTER JOIN dolt_commit_ancestors AS dca
+        ON dc.commit_hash = dca.commit_hash
+    WHERE dc.commit_hash = HASHOF('HEAD')
+    ORDER BY date DESC
+    LIMIT 1;
+SQL
+    echo $status
```

Comment:
Get rid of theses. They are trapped if the test passes and you get output if it fails. echos are useless.

Reply:
fixing